### DISCUSSION TITLE
Replace Pagination Widget with More button

### DISCUSSION THREAD

**Discussion by mikebhart** [algolia/instantsearch#4507] [General]
Hi there, I'm using the instantsearch js pagination widget. Is there a way of replacing it with just a more button.  So no page numbers, no next prev etc, just one button to go to the next page.

**Comment by Haroenv** [comment]
- if the pages should be shown together, change `hits` for `infiniteHits`
- if pagination is expected, use `connectPagination` and `refine(currentRefinement + 1)`, you can look how the real pagination widget works

**Comment by mikebhart** [comment]
I have this working thankyou.  Currently the next button will replace the current generated hits with the new page set, is there a way of tailing on the end of the existing hit items rather than replace the old set.  Like a typical load more button would work.

**Reply by Haroenv** [reply]
yes, that's infiniteHits, as I said in the first comment...
